packages/flutter/test/rendering/viewport_test.dart

```diff
@@ -837,30 +837,21 @@ void main() {
             center: centerKey,
             reverse: true,
             slivers: <Widget>[
-              SliverList(
-                delegate: SliverChildListDelegate(
-                  List<Widget>.generate(
-                    10,
-                    (int index) => SizedBox(height: itemHeight, child: Text('Item ${-index - 1}')),
-                  ),
-                ),
+              SliverList.builder(
+                itemCount: 10,
+                itemBuilder: (BuildContext context, int index) {
+                  return SizedBox(height: itemHeight, child: Text('Item ${-index - 1}'));
+                },
```

Comment:
I should've used `SliverList.list` here to exactly match the previous behavior. However,  whether these widgets are built lazily or eagerly does not appear to affect the test, and, using `SliverList.builder` reduces boilerplate. Let me know if you'd like me to switch to `SliverList.list` though!